src/annotator/guest.js

```diff
@@ -143,22 +143,11 @@ export default class Guest {
     /** @type {Anchor[]} */
     this.anchors = [];
 
-    // Setup the document type-specific integration consisting of metadata extraction,
-    // anchoring module and logic to respond to activity (eg. scrolling) in the page.
-    if (config.documentType === 'pdf') {
-      this.integration = new PDFIntegration(this);
-    } else {
-      const documentMeta = new DocumentMeta();
-      this.integration = {
-        anchor: htmlAnchoring.anchor,
-        contentContainer: () => this.element,
-        describe: htmlAnchoring.describe,
-        destroy: () => {},
-        fitSideBySide: () => false,
-        getMetadata: () => Promise.resolve(documentMeta.getDocumentMetadata()),
-        uri: () => Promise.resolve(documentMeta.uri()),
-      };
-    }
+    /** @type {Integration} */
+    this.integration =
+      config.documentType === 'pdf'
+        ? new PDFIntegration(this)
+        : new HTMLIntegration(this.element);
```

Comment:
In future it will probably sense to pass in the integration implementation rather than have `Guest` know how to construct it.